apps/fares/lib/fare_info.ex

```diff
@@ -5,30 +5,22 @@ defmodule Fares.FareInfo do
 
   alias Fares.Fare
 
-  @doc "Load fare info from a CSV file."
-  @september_1_2020 1_598_932_800
   @september_1_2020_modes ["subway", "local_bus", "inner_express_bus", "outer_express_bus"]
 
+  @doc "Load fare info from a CSV file."
   @spec fare_info() :: [Fare.t()]
-  @spec fare_info(integer) :: [Fare.t()]
-  def fare_info(now \\ System.system_time(:second)) do
-    now
-    |> filename()
+  def fare_info() do
+    "priv/fares-sept1.csv"
     |> fare_data()
-    |> Enum.flat_map(&mapper(&1, now))
+    |> Enum.flat_map(&maybe_combine_and_map/1)
     |> Enum.concat(free_fare())
     |> split_reduced_fares()
   end
 
-  @spec filename(integer) :: Path.t()
-  defp filename(now) when now >= @september_1_2020, do: "priv/fares-sept1.csv"
-
-  defp filename(_), do: "priv/fares-july1.csv"
-
   @doc "Combines paper and plastic fare into a single price for certain modes"
-  @spec mapper([String.t()], integer) :: [Fare.t()]
-  def mapper([mode | _] = data, now)
-      when mode in @september_1_2020_modes and now >= @september_1_2020 do
+  @spec maybe_combine_and_map([String.t()]) :: [Fare.t()]
+  def maybe_combine_and_map([mode | _] = data)
```

Comment:
There are some very simple steps I can code in here to avoid this clause. Just go into the mappers and manually combine/merge these things for these modes. Mind if I do that real quick?

Reply:
I'll just go and do this later in the week as a cleanup.